### DISCUSSION TITLE
Manually placing hideouts in Eden ?

### DISCUSSION THREAD

**Discussion by Manthhys** [Vdauphin/HeartsAndMinds#1256] [Question and answer] [answered]
Hi there, did anyone play around to see if it is possible to manually design hideouts for exemple in buildings or occupied FOBs in H&M ?

I'd like to know if I could manually place an item that represent the hideout that generates the ennemy garrison and must be destroyed by the players ? For example by using the actual variable name used for the item we have to destroy in H&M vanilla.

**Comment by Vdauphin** [comment]
Hello,

It is not easy, you need to make sure this https://github.com/Vdauphin/HeartsAndMinds/blob/master/%3DBTC%3Dco%4030_Hearts_and_Minds.Altis/core/fnc/mil/create_hideout.sqf#L44-L74 detect your objects

See here also
https://github.com/Vdauphin/HeartsAndMinds/issues/643

Cheers

**Reply by Manthhys** [reply]
Thanks, I'm going to look into it ;)

**Comment by Raz0rLegend** [comment]
@Manthhys Hello ! 
Did you find a way to do this ? I look into it too. But, i don't know where i need to change something to whrite the 5 positions of the Hideout.

**Reply by Manthhys** [reply]
Hey ! Havent had the time yet to get elbow-deep into the code, I'll post here if I get something working ;)

**Reply by Raz0rLegend** [reply]
I had'nt find it yet too. I do not understand what the code have to find inside the Eden. I know the code search a safe place to put the hideout. But If I place them inside Eden. What should I place ?

**Reply by Manthhys** [reply]
From what I got, the hideout generator finds cities and generate hideouts around them, so my guess is one should modifiy the code to detect your eden object as a point to spawn hideouts.

**Reply by Raz0rLegend** [reply]
Like the red flag for example. 
If I put a red flag the mission knows it has to put the hideout here.

**Comment by Vdauphin** [comment, marked as answer]
Hello, 

You may want to take a look to this https://github.com/Vdauphin/HeartsAndMinds/pull/1529
After `btc_hideout_cityID` defined, you will be allow to slightly move hideout around the city you choose with Zeus

Don't move the hideout away from the city otherwise the city will be out of range of the hideout and nothing will spawn inside the city

Cheers
